maestro-ios/src/main/java/ios/idb/TextInputUtil.kt

```diff
@@ -195,7 +195,7 @@ internal object TextInputUtil {
         '*' to keyPressShiftedToEvents(37),
         '(' to keyPressShiftedToEvents(38),
         ')' to keyPressShiftedToEvents(39),
-        '"' to keyPressShiftedToEvents(45),
+        '_' to keyPressShiftedToEvents(45),
```

Comment:
isn't underscore code 95?

Reply:
Sorry @dmitry-zaitsev .. I can't test. The truth is that I was doing a flow and I noticed that the underscore is replaced by "". I found this issue and decided to PR it.

If you want I can change it to 95 but I can't really test it.

Reply:
@dmitry-zaitsev Okay, I ended up change it. Please, take another look. Thanks.

Reply:
I checked your changes and your original version was correct. It should be 45

Sorry for the confusion